modules/ROOT/pages/apikit-odatav4-extension-reference.adoc

```diff
@@ -226,39 +226,40 @@ Default configuration
 [cols="1a,2a,3a,4a,5a", options="header"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | Name of the configuration to use | | *x*
-| Entity Set Name | String | Table name in the database for the requested entity set |  | *x*
-| Database Schema | String | Database schema name to which the entity set belongs. If not empty, it's prefixed to the table name in the `WHERE` clause |  |
-| Page Size | Number |  |  |
-| Config Ref | ConfigurationProvider | Name of the configuration used to execute this component |  | *x*
-| Streaming Strategy |
+| Config Ref | ConfigurationProvider | Name of the configuration used to execute this component |  | Yes
+| Configuration | String | Name of the configuration to use | | Yes
+| Database Schema | String | Database schema name to which the entity set belongs. If not empty, it's prefixed to the table name in the `WHERE` clause |  | No
+| Entity Set Name | String | Table name in the database for the requested entity set |  | Yes
+| Entity Type Fields | Array of String | List of property fields from the Entity type |  | Yes
+| Entity Type Keys | Object | Map of entity key names and values |  | Yes
+| Entity Type Name | String | Entity Data Model specifies the entity type name |  | Yes
+| Page Size | Number |  |  | No
+| searchFieldsAsType | Object | Select if the DB columns is typecasted to TEXT when using the `$search` operation | | No
```

Comment:
Not sure if this description is auto-generated. The description of this field in code is `Option to typecast fields for search transformation`

The description is not accurate. Consider rephrasing as
Data type to which fields are type-casted for `$search` operation. If empty, all fields are typecast as TEXT by default
cc @ruturajgujar

Reply:
Thanks to @ruturajgujar, please update the description as follows

Data type to which fields are type-casted for $search operation. If empty, no type-casting is performed.

Reply:
@Crispy-Salesforce I have updated the field description as per Aby's suggestion. Can you please update the reference doc file?

Reply:
Hi, team! Thanks for your feedback!